packages/klighd-core/src/options/render-options-registry.ts

```diff
@@ -173,6 +231,8 @@ export class TitleScalingFactor implements RangeOption {
     }
     readonly stepSize = 0.01
     readonly initialValue: number = TitleScalingFactor.DEFAULT
+    readonly renderCategory: string = SmartZoom.ID
+    readonly description = 'Factor to scale region titles compared to their original size.\nIf set to 1 a region title will never exceed their original size.'
```

Comment:
If set to 1 a region title will never be smaller than its original apparent size. This wording sounds as if the text will always be smaller than original.

Reply:
But if the space does not permit it, the region title is drawn smaller. As I see it we specify an upper bound here. I will change it to will be their original size if the space permits it.